src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -309,10 +309,7 @@ class DataResourceDataProvider implements IGridDataProvider {
 		return serializer.handleSerialization(this.documentUri, format, (filePath) => this.doSerialize(serializer, filePath, format, selection));
 	}
 
-	private doSerialize(serializer: ResultSerializer, filePath: URI, format: SaveFormat, selection: Slick.Range[]): Promise<SaveResultsResponse | undefined> {
-		if (!this.canSerialize) {
-			return Promise.resolve(undefined);
-		}
+	getSerializeRequestParams(serializer: ResultSerializer, filePath: URI, format: SaveFormat, selection: Slick.Range[]) {
```

Comment:
Don't remove private - tests shouldn't need to be concerned with underlying implementation. Instead you should be looking at testing the `serializeResults` function